src/main/java/net/revelc/code/impsort/ImpSort.java

```diff
@@ -216,12 +216,17 @@ private static void convertAndAddImport(LinkedHashSet<Import> allImports, List<N
   private static Set<String> tokensInUse(CompilationUnit unit) {
 
     // Extract tokens from the java code:
-    Stream<String> typesInCode =
-        unit.getTypes().stream().map(TypeDeclaration::getTokenRange).filter(Optional::isPresent)
-            .map(Optional::get).filter(r -> r != TokenRange.INVALID).flatMap(r -> {
-              // get all JavaTokens as strings from each range
-              return StreamSupport.stream(r.spliterator(), false);
-            }).map(JavaToken::asString);
+    Stream<Node> packageDecl =
+        unit.getPackageDeclaration().isPresent()
+            ? Stream.of(unit.getPackageDeclaration().get()).map(PackageDeclaration::getAnnotations)
+                .flatMap(NodeList::stream)
+            : Stream.empty();
+    Stream<String> typesInCode = Stream.concat(packageDecl, unit.getTypes().stream())
```

Comment:
Not sure how I feel about concat here, since this gets us closer to the code we had before it was recently cleaned up, when it was much harder to read (described at nightmarish in 6a760b37c4e4cd251cf5a811083844963f5ac7da).

If you have any ideas here for alternate implementations that could make this slightly more readable, I'm open to that. Otherwise, I'll defer any readability cleanup to a later task.